### PR TITLE
Test filters with shortcuts

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/cheminfo/nmrium/issues/1694

### CODE REVIEW

File: test-e2e/panels/filters.test.ts [hunk 1]
```diff
@@ -33,22 +36,51 @@ async function fourierTransformFilter(nmrium: NmriumPage) {
...
 }
 
-async function phaseCorrectionFilter(nmrium: NmriumPage) {
-  await nmrium.clickTool('phaseCorrection');
-  await nmrium.page.fill('data-test-id=input-ph1', '-100');
-  await nmrium.page.fill('data-test-id=input-ph0', '-104');
-
-  // input debounce for 250ms
-  await nmrium.page.waitForTimeout(250);
+async function phaseCorrectionFilter(
```
Comment (lpatiny) [inline test-e2e/panels/filters.test.ts:39]: @targos I would like your opinion about where to put those functions. If I understood well your idea is to move the code as a method of NmriumPage (just the action, not the expect) ?

The number of methods (and associated code) will become quite big so I would also suggest there is one file per big method of NmriumPage instance.

Comment (targos) [reply test-e2e/panels/filters.test.ts:39]: I agree with that. In this case for example I would add a method named `applyPhaseCorrection(options)` to the class.

Comment (wadjih-bencheikh18) [reply test-e2e/panels/filters.test.ts:39]: only applyPhaseCorrection functions? what about others?

Comment (targos) [reply test-e2e/panels/filters.test.ts:39]: It's up to you. anything that may be done in more than one test can be there.

Comment (wadjih-bencheikh18) [reply test-e2e/panels/filters.test.ts:39]: okey perfect thanks

Comment (wadjih-bencheikh18) [reply test-e2e/panels/filters.test.ts]: > anything that may be done in more than one test can be there.

i added applyPhaseCorrection , in the next PR if i reused any other function i will add it in the class

**Pull request merged**